backend/src/main/java/de/bund/digitalservice/ris/caselaw/adapter/database/jpa/OriginalXmlDTO.java

```diff
@@ -0,0 +1,45 @@
+package de.bund.digitalservice.ris.caselaw.adapter.database.jpa;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.MapsId;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import java.time.Instant;
+import java.util.UUID;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder(toBuilder = true)
+@Entity
+@Table(name = "original_xml", schema = "incremental_migration")
+public class OriginalXmlDTO {
```

Comment:
question: Why OriginalXmlDTO and not MigrationDTO?

Reply:
Good question! We name the database entities after the table names (`original_xml`)  and not the domain models